indico/modules/events/roles/templates/_roles.html

```diff
@@ -83,6 +83,11 @@
                     <div class="empty">{% trans %}There are no members yet.{% endtrans %}</div>
                 {% endif %}
                     <div class="toolbar f-j-end">
+                         <button class="i-button highlight text-color icon-upload"
+                                data-href="{{ url_for('.add_members_import_csv', role) }}"
+                                data-update="#event-role-{{ role.id }}"
+                                data-ajax-dialog>{% trans %}Import users from CSV{% endtrans %}
```

Comment:
`from CSV file`?

Reply:
`from CSV` would be good, I think

Reply:
an example of what we have in a few other places
```html
<a class="i-button"
    title="{% trans %}Import contributions from CSV file{% endtrans %}"
    data-href="{{ url_for('.contributions_import', event) }}"
    data-title="{% trans %}Import contributions from CSV File{% endtrans %}"
    data-ajax-dialog>
    {%- trans %}Import{% endtrans -%}
</a>
```